src/explorer/WebJobsTreeItem.ts

```diff
@@ -19,19 +19,27 @@ export class WebJobsTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     }
 
     public get iconPath(): { light: string, dark: string } {
-        return {
-            light: path.join(resourcesPath, 'light', 'WebJobs_color.svg'),
-            dark: path.join(resourcesPath, 'dark', 'WebJobs_color.svg')
-        };
+        return this.root.client.isLinux ?
+            {
+                light: path.join(resourcesPath, 'light', 'WebJobs_grayscale.svg'),
+                dark: path.join(resourcesPath, 'dark', 'WebJobs_grayscale.svg')
+            } :
+            {
+                light: path.join(resourcesPath, 'light', 'WebJobs_color.svg'),
+                dark: path.join(resourcesPath, 'dark', 'WebJobs_color.svg')
+            };
     }
 
     public hasMoreChildrenImpl(): boolean {
         return false;
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzureTreeItem<ISiteTreeRoot>[]> {
-        const jobList: webJob[] = <webJob[]>await this.root.client.kudu.jobs.listAllJobs();
+        if (this.root.client.isLinux) {
+            return [new GenericTreeItem<ISiteTreeRoot>(this, { label: 'WebJobs are not available for Linux Apps.', contextValue: 'webJobNA' })];

```

Comment:
Can you add a link to docs for this? Just want to make sure we're not blocking something that might work a week or two from now.

Actually it might be better to do a try/catch and add this logic in the catch block so that we will work with webJobs if/when that ever happens

Reply:
Very smort